src/components/v2/VoteSummary/index.tsx

```diff
@@ -0,0 +1,101 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
Since this component will only be used on the Proposal page it can live there. I have pushed a branch I am working on for setting up the skeleton if you want to make the PR into that branch `vote-proposal-skeleton`

Reply:
Looks like we created a new directory. This skeleton for this component is called Votes (We can change the name). It is setup in the layout of the page as well (`src/pages/Proposal/index.tsx`).

We should drop this component there so it is rendered on the page with mock data until the page is wired up